netsplit.c

```diff
@@ -237,7 +237,7 @@ void show_split_networks_v6(unsigned split_prefix, const struct ip_info_st *info
 		count++;
 	}
 
-	dist_printf("\nTotal:  \t", "%u\n", count);
-	dist_printf("Hosts/Net:\t", "%s\n", ipv6_prefix_to_hosts(buf, sizeof(buf), split_prefix));
+	color_printf(ctl, KMAG, "\nTotal:  \t", "%u\n", count);
+	color_printf(ctl, KMAG, "Hosts/Net:\t", "%s\n", ipv6_prefix_to_hosts(buf, sizeof(buf), split_prefix));
 }
```

Comment:
r+
I like this change.
A question is why do you bother with such a change for a command line tool? Are you using parts as library?